pkgs/by-name/az/azure-cli/extensions-manual.nix

```diff
@@ -42,6 +42,20 @@
     ];
     meta.maintainers = with lib.maintainers; [ giggio ];
   };
+  ssh = mkAzExtension rec {
+    pname = "ssh";
+    version = "2.0.5";
+    url = "https://azcliprod.blob.core.windows.net/cli-extensions/ssh-${version}-py3-none-any.whl";
+    sha256 = "80c98b10d7bf1ce4005b7694aedd05c47355456775ba6125308be65fb0fefc93";
+    description = "SSH into Azure VMs using RBAC and AAD OpenSSH Certificates";
+    propagatedBuildInputs =
+      (with python3Packages; [
+        oras
+        oschmod
+      ])
+      ++ [ mycli ];
```

Comment:
Is `mycli` needed? The extension doesn't seem to require it.

Reply:
I think it is left over from another manual extension which I copied from. Thanks for pointing it out!